src/main/java/com/smartbear/mqttsupport/connection/dialog/EditConnectionDialog.java

```diff
@@ -674,7 +677,7 @@ protected boolean handleOk() {
         return true;
     }
 
-    private void activateTab(String tabName) {
+    void activateTab(String tabName) {
```

Comment:
it is a good practice to have access modifiers added to methods in java. Please add a suitable modifier (protected or public)

Reply:
I had to give up on the `private` access modifier, so those methods are available for testing. By not specifying a modifier I applied `default` / `package private` modifier. As I understand we want to have as "private" modifier as possible. Given that `default` / `package private` modifier is more "private" than `protected` or `public` I choose this one. I have based my implementation on this: https://www.javatpoint.com/access-modifiers 

Does it make sense? Or should I change it to something different?